addon/components/frost-date-picker.js

```diff
@@ -1,5 +1,7 @@
 import Ember from 'ember'
 
-export default Ember.Component.extend({
+const {Component} = Ember
```

Comment:
Yeah, it's not full ES6 style yet, but it's coming - @rox163 have we mentioned the new `const` convention in Slack yet?